tests/drivers/dma/loop_transfer/boards/adp_xc7k_ae350.conf

```diff
@@ -0,0 +1 @@
+CONFIG_CACHE_ENABLE=n
```

Comment:
uhm, it's a bit too late but having this symbol is wrong since it overlaps with the cache symbols we already have in place in Zephyr. Can you ask some Andes folk to remove it?

Reply:
I got it!
We use the CONFIG_ICACHE and CONFIG_DCACHE to replace it now.